spidermon/contrib/actions/slack/__init__.py

```diff
@@ -203,8 +203,8 @@ def run_action(self):
                 to=self.recipients, text=message, attachments=attachments
             )
         else:
-            print("message:", message)
-            print("attachments:", attachments)
+            logging.info(message)
```

Comment:
Use `logger.info(message)` and include `logger = logging.getLogger(__name__)` in the beginning of the module, to make the logger module-based (do the same in any other place we have logs like this).

Reply:
Done!
Please let me know if the way I did conforms with the guidelines because I didn't find any example on PEP08 or Python logging guidelines.